atomic_reactor/inner.py

```diff
@@ -312,6 +313,9 @@ def base_image_inspect(self):
             self._base_image_inspect = self.builder.tasker.inspect_image(self.builder.base_image)
         return self._base_image_inspect
 
+    def throw_cancelled_build_exception(self, *args, **kwargs):
+        raise RuntimeError("Build was cancelled")
```

Comment:
Can we define and use a specific exception for this? Perhaps `CancelError`? That way plugins can perform any specific cancel operations they need to by catching it.